portal/src/components/Popup/CreateAccount/CreateAccount.component.js

```diff
@@ -0,0 +1,52 @@
+import Popup from '~/components/Popup'
+import { isNil } from 'lodash'
+export default {
+  name: 'create-account',
+  props: ['is-show', 'close', 'user'],
+  components: {
+    Popup
+  },
+  data() {
+    return {
+      is_submitting: false
+    }
+  },
+  methods: {
+    onCreateAccount() {
+      this.is_submitting = true
+      this.$store
+        .dispatch('postUser')
+        .then(() => {
+          setTimeout(() => {
+            let authFlowToken = this.$store.getters.auth_flow_token
+            if (!isNil(authFlowToken)) {
```

Comment:
isNil is hier wel beetje overbodig in mijn ogen `!authFlowToken` kan ook.

Reply:
tja, ik had deze method overgenomen van ergens anders. maar dan kan het natuurlijk geen kwaad om er even goed naar te kijken. 😉

Reply:
maar dan moet het authFlowToken zijn, want er staat als het NIET nil is.